.travis.yml

```diff
@@ -21,27 +21,27 @@ matrix:
   fast_finish: true
 
   include:
-    - php: 7.1
+    - php: 7.3
       env: DB=pgsql db_dsn='postgres://postgres@127.0.0.1/cakephp_test'
 
-    - php: 7.2
+    - php: 7.3
       env: DB=sqlite db_dsn='sqlite:///:memory:'
 
-    - php: 5.6
+    - php: 7.2
       env: PREFER_LOWEST=1
 
-    - php: 7.2
+    - php: 7.3
       env: CHECKS=1 DEFAULT=0
 
-    - php: 7.2
+    - php: 7.3
       env: CODECOVERAGE=1 DEFAULT=0
 
 before_script:
   - if [[ $PREFER_LOWEST != 1 ]]; then composer install --prefer-source --no-interaction ; fi
-  - if [[ $PREFER_LOWEST == 1 ]]; then composer update --prefer-dist --no-interaction --prefer-lowest --prefer-stable ; fi
   - if [[ $PREFER_LOWEST == 1 ]]; then composer require --dev dereuromark/composer-prefer-lowest; fi
+  - if [[ $PREFER_LOWEST == 1 ]]; then composer update --prefer-dist --no-interaction --prefer-lowest --prefer-stable ; fi
```

Comment:
Don't change the order.

Reply:
There was an issue with the 'semver' requirement from 'dereuromark/composer-prefer-lowest', which is not encountered if requested with '--prefer-lowest --prefer-stable'.

Reply:
Yeah I'm conversing with @dereuromark and he said use ^8.5 he knows there's issues with phpunit and semver

Reply:
Just saw this, reverted sequence and used ^8.5